contracts/src/libraries/LPMath.sol

```diff
@@ -28,6 +28,103 @@ library LPMath {
     ///      succeed.
     uint256 internal constant SHARE_PROCEEDS_TOLERANCE = 1e14;
 
+    struct PresentValueParams {
+        uint256 shareReserves;
+        int256 shareAdjustment;
+        uint256 bondReserves;
+        uint256 vaultSharePrice;
+        uint256 initialVaultSharePrice;
+        uint256 minimumShareReserves;
+        uint256 minimumTransactionAmount;
+        uint256 timeStretch;
+        uint256 longsOutstanding;
+        uint256 longAverageTimeRemaining;
+        uint256 shortsOutstanding;
+        uint256 shortAverageTimeRemaining;
+    }
+
+    struct DistributeExcessIdleParams {
+        PresentValueParams presentValueParams;
+        uint256 startingPresentValue;
+        uint256 activeLpTotalSupply;
+        uint256 withdrawalSharesTotalSupply;
+        uint256 idle;
+        int256 netCurveTrade;
+        uint256 originalShareReserves;
+        int256 originalShareAdjustment;
+        uint256 originalBondReserves;
+    }
+
+    /// @dev Calculates the initial reserves. We solve for the initial reserves
+    ///      by solving the following equations simultaneously:
+    ///
+    ///      (1) c * z = c * z_e + p_target * y
+    ///
+    ///      (2) p_target = ((mu * z_e) / y) ** t_s
+    ///
+    ///      where p_target is the target spot price implied by the target spot
+    ///      rate.
```

Comment:
It looks to me like the formula for (2) is different than [what is stated in the hackmd](https://hackmd.io/yMNxotdqRKSuNrac9w8uFw?view#Zeta-and-Bond-Reserves-for-Initialization).

Is this due to a simplification?

Reply:
The effective share reserves, $z_e$, is defined as $z_e = z - \zeta$, where $z$ denotes the shares reserves and $\zeta$ denotes the share adjustment. This is typically how I write the price, and it's just a matter of personal preference IMO (I didn't write the HackMD).